internal/process/provisioning/resolve_creds_hap_selection_migration_test.go

```diff
@@ -0,0 +1,226 @@
+package provisioning
```

Comment:
At the office we discussed running the same test once for old implementation, storing data and result, and using this data to test regressively new implementation for some time (till we stop changing extensively this part of code).
This approach would let to remove old implementation earlier.